tests/doubles.py

```diff
@@ -3,40 +3,36 @@
 # for complete details.
 
 
-from cryptography import utils
 from cryptography.hazmat.primitives import hashes, serialization
 from cryptography.hazmat.primitives.asymmetric import padding
 from cryptography.hazmat.primitives.ciphers import CipherAlgorithm
 from cryptography.hazmat.primitives.ciphers.modes import Mode
 
 
-@utils.register_interface(CipherAlgorithm)
-class DummyCipherAlgorithm(object):
+class DummyCipherAlgorithm(CipherAlgorithm):
     name = "dummy-cipher"
     block_size = 128
     key_size = None
 
 
-@utils.register_interface(Mode)
-class DummyMode(object):
+class DummyMode(Mode):
     name = "dummy-mode"
 
     def validate_for_algorithm(self, algorithm: CipherAlgorithm) -> None:
         pass
 
 
-@utils.register_interface(hashes.HashAlgorithm)
-class DummyHashAlgorithm(object):
+class DummyHashAlgorithm(hashes.HashAlgorithm):
     name = "dummy-hash"
     block_size = None
-    digest_size = None
+    digest_size = None  # type: ignore[assignment]
```

Comment:
Is there a reason we don't have a real value here? Or that the interface is wrong?